guides/Codable.md

```diff
@@ -0,0 +1,102 @@
+## I want to make my types conform to `Encodable` or `Decodable`
+
+### [Encodable Template](https://github.com/krzysztofzablocki/Sourcery/blob/master/Templates/Templates/Encodable.stencil)
+### [Decodable Template](https://github.com/krzysztofzablocki/Sourcery/blob/master/Templates/Templates/Encodable.stencil)
+
+#### Available annotations
+
+
+* `skipCoding` - skips decoding and encoding
```

Comment:
I believe this annotation should be applied only for variables (should be mentioned in description) and only for encoding, so should be `skipEncoding`. For types we wouldn't need it until inheritance is implemented.